app/src/main/java/app/grapheneos/camera/ui/SettingsDialog.kt

```diff
@@ -77,6 +77,10 @@ class SettingsDialog(val mActivity: MainActivity) :
     private var videoQualitySetting: View
     private var timerSetting: View
 
+    var smRadioGroup: RadioGroup
```

Comment:
These names are too short. Also, `var` should be replaced with `val` whenever possible.

Reply:
I'll make these changes (for the `SettingsDialog` class as a whole) to the main branch of our main repository separately once we're done merging these changes. Thanks a lot for making a note of this @muhomorr!